__tests__/get-api-categories.test.js

```diff
@@ -0,0 +1,42 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed")
+const connection = require("../db/connection");
+const testData = require("../db/data/test-data/index")
+
+beforeEach(() => {
+    return seed(testData);
+})
+
+afterAll(() => {
+    return connection.end()
+})
+
+
+describe('Testing the /api/categories', () => {
+    test('testing that get api/categories returns the right information and returns status 200', () => {
+        return request(app)
+        .get("/api/categories")
+        .expect(200)
+            .then((result) => {
+                const categories = result.body.categories
+                expect(categories.length).toBe(4)
+                
+                expect(categories[0]).toHaveProperty('slug')
```

Comment:
This is only checking the first item in the array. Can you change this to check each item in the array looks how we expect?